alt-tab-macos/logic/Keyboard.swift

```diff
@@ -57,10 +59,14 @@ func keyboardHandler(proxy: CGEventTapProxy, type: CGEventType, event_: CGEvent,
                     return dispatchWork(application, true, { application.cycleSelection(1) })
                 } else if isLeftArrow && application.appIsBeingUsed {
                     return dispatchWork(application, true, { application.cycleSelection(-1) })
+                } else if isQ && application.appIsBeingUsed {
+                    return dispatchWork(application, true, { application.quitTargetApp() })
+                } else if isW && application.appIsBeingUsed {
+                    return dispatchWork(application, true, { application.closeTarget() })
                 } else if type == .keyDown && isEscape {
                     return dispatchWork(application, false, { application.hideUi() })
                 }
-            } else if isMetaChanged && !isMetaDown {
+            } else if isMetaChanged && !isMetaDown && application.appIsBeingUsed {
```

Comment:
I'm confused as to why you added this condition? Did you run into a corner-case that required this?

Reply:
I could probably have made a separate commit for this. It's just, that the application got triggered when only the modifier got pressed. And we don't have to check if the application is being used like here: https://github.com/lwouis/alt-tab-macos/blob/c69123b3d4d7b6884f3f6ebd7848a4d6d8e579e5/alt-tab-macos/ui/Application.swift#L53-L56

I can definitely change this back if you like.

Reply:
I have thrown this out for the moment